### PR TITLE
[Java][UPDT]: changed directory structure

### PR DESCRIPTION
**Pull request by choudharyvikram**

### CODE REVIEW

**Comment by choudharyvikram** [comment]
We should have some standards in place for things. Like commit message, filenames, etc.
I have added few examples in ReadMe

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # BasicProgrammingQuestions
 Interview Ready Basic Programming Questions
+
+### Commits Format
+[branch_name][commit_type]: message
+#### Commit Types:
+    QUE: Question(s) was added
+    UPDT: Update was made in the repo
+    BUG: Bug fix
```
Comment (choudharyvikram) [inline README.md:9]: you can add/remove types here. or could suggest any other method that you know of

**Review by niveditapatekar27 — APPROVED** [review]

**Review by niveditapatekar27 — COMMENTED** [review]
Reviewed changes

**Comment by niveditapatekar27** [comment]
Reviewed changes

**Pull request merged**